<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyoGuiClass</class>
 <widget class="QMainWindow" name="MyoGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyoGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="glwidget" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>570</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="VideoWidget" name="videowidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="CameraWidget" name="camerawidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QSlider" name="progressSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>640</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>531</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="AudioWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <zorder>camerawidget</zorder>
   <zorder>videowidget</zorder>
   <zorder>glwidget</zorder>
   <zorder>stopButton</zorder>
   <zorder>startButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>progressSlider</zorder>
   <zorder>playButton</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header>camerawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioWidget</class>
   <extends>QWidget</extends>
   <header>audiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
